src/AutoRest.CSharp/MgmtTest/Generation/Mock/MgmtMockTestBaseWriter.cs

```diff
@@ -44,6 +48,75 @@ public virtual void Write()
             }
         }
 
+        public virtual void WriteSample(MockTestCase testCase)
+        {
+            generatingSample = true;
+            Regex pattern = new Regex(@".*(Azure.ResourceManager.\w+)/.*");
+            Match match = pattern.Match(Configuration.OutputFolder);
+            if (match.Success)
+            {
+                _writer.additionalComments.Add($"// Need to install {match.Groups[1].Value} v1.0.0 to execute below sample code");
+            }
+            _writer.addUsingRegion = true;
```

Comment:
If we need to change something in our code writer, please use a property instead. Changing a public field is always not recommended